build.sbt

```diff
@@ -1,9 +1,72 @@
 import sbtcrossproject.{crossProject, CrossType}
 
-val Scala212 = "2.12.10"
-val Scala213 = "2.13.1"
+val Scala212 = "2.12.12"
+val Scala213 = "2.13.4"
 val Scala3 = "3.0.0-M1"
 
+ThisBuild / crossScalaVersions := Seq(Scala213, Scala212)
+ThisBuild / scalaVersion := Scala213
+
+ThisBuild / githubWorkflowJavaVersions := Seq("adopt@1.11")
+
+ThisBuild / githubWorkflowBuildMatrixAdditions += "platform" -> List("jvm", "js")
+
+val JvmCond = s"matrix.platform == 'jvm'"
+val JsCond = s"matrix.platform == 'js'"
+
+val Scala2Cond = s"matrix.scala != '$Scala3'"
+val Scala3Cond = s"matrix.scala == '$Scala3')"
+
+val Scala212Cond = s"matrix.scala == '$Scala212'"
+
+ThisBuild / githubWorkflowBuild := Seq(
+  WorkflowStep.Sbt(List("js/checkCI"), name = Some("Validate JavaScript"), cond = Some(JsCond)),
+  WorkflowStep.Sbt(List("jvm/checkCI"),
+                   name = Some("Validate JVM"),
+                   cond = Some(JvmCond + " && " + s"matrix.scala != '$Scala212'")
+  ),
+  WorkflowStep.Use("actions",
+                   "setup-python",
+                   "v2",
+                   name = Some("Setup Python"),
+                   params = Map("python-version" -> "3.x"),
+                   cond = Some(JvmCond + " && " + Scala212Cond)
+  ),
+  WorkflowStep.Run(List("pip install --user codecov"),
```

Comment:
@larsrh @johnynek  sorry for the failures. I copied the `--user` switch from the `.travisci` for parity but perhaps its not working in this context as it seems to be causing an issue in this patch. In the [frameless](https://github.com/typelevel/frameless/blob/master/.github/workflows/ci.yml#L60) repo I didn't use this switch and the step succeeded. Perhaps we should do the same here?